specification/glossary.md

```diff
@@ -0,0 +1,50 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* (or API compatible) library in use.  
+See [Library](data-resource-semantic-conventions.md#library)
+
+<a name="instrumented_library"></a>
+
+## Instrumented Library
+
+Denotes the library whose "actions" (e.g. called methods/functions, operations, ...)
+result in OpenTelemetry signals (traces, metrics, logs).  
+The actual OpenTelemetry API calls can happen inside this library but might as well
+happen in a different one (the [Instrumenting Library](#instrumenting_library)).
+
+Example: `org.mongodb.client`.
+
+<a name="instrumenting_library"></a>
+
+## Instrumenting Library
+
+Denotes the library which provides the instrumentation *for* a given library (i.e.
+the [Instrumented Library](#instrumented_library)). Hence, *Instrumented Library*
+and *Instrumenting Library* can be identical if a library has built-in OpenTelemetry
+instrumentation, but they don't necessarily have to.
+
+Example: `io.opentelemetry.contrib.mongodb`.
+
+Synonyms: *Instrumentation Library*, *Integration*.
+
+<a name="name"></a>
+
+## Tracer Name / Meter Name
+
+This refers to the `name` and (optional) `version` arguments specified when
```

Comment:
Does it really refer to the version too? I think we have to concede that when someone is using "tracer/meter name" it is unclear whether the version is included.

Reply:
Leaving that in, since it was demanded by previous reviewers and upvoted (liked).